File: 1598. Crawler Log Folder/1598.Crawler Log Folder.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        opreations = []
        for op in logs:
            if op == '../':
                if len(opreations) > 0:
                    opreations.pop()
            elif op == './':
                continue
            else:
                opreations.append(op)

        return len(opreations)

File: 1598. Crawler Log Folder/1598.Crawler Log Folder.js
/**
 * @param {string[]} logs
 * @return {number}
 */
var minOperations = function(logs) {
    let operations = [];
    for(let op in logs) {
        if(logs[op] === "../") {
            if(operations.length > 0)
                operations.pop();
        } else if(logs[op] === "./") {
        } else {
            operations.push(logs[op]);
        }

    }
    return operations.length;
};


console.log(minOperations(["d1/","d2/","../","d21/","./"])); // 2